### ISSUE TITLE
Hyperparameters for StanfordORB

### ISSUE THREAD

**Issue by qbhan** [stopaimme/GI-GS#1]
Hi.

First, thank you for your excellent work on simulating global illumination for GS-based inverse rendering! Your contributions are greatly appreciated.

I'm particularly interested in real-world inverse rendering datasets, and I noted that results for StanfordORB were posted on arXiv. I've attempted to run GI-GS on the StanfordORB dataset myself, but my initial results suggest that some hyperparameter tuning may be required.

Could you please share the specific hyperparameters you used when running GI-GS on the StanfordORB dataset?

Additionally, during your research, did you observe any significant optimization discrepancies between synthetic and real-world datasets? If so, could you elaborate on these differences?

Any help, insights, or tips you could provide would be highly valuable.

Thank you again for your time and assistance.

![Image](https://github.com/user-attachments/assets/a2c81333-9408-46e7-9510-a4e090cbb2d4)

**Comment by stopaimme** [comment]
### About the artifact at the edges of objects.
Since the images in the Stanford-ORB dataset contain the background, so during the training, we calculate the **masked loss** using the mask given in the dataset. It will cause the artifacts in the background near the edges of objects. In the paper, we use mask to obtain the clean image. Maybe you can set the **"pad_normal"** in render function to "False" to reduce the artifacts (I'm not sure about it, since I forget the setting I used for the Stanford-ORB dataset). But I am sure that the main cause is the mask used during training.

For synthetic dataset, like Tenso-IR, since the images in the dataset do not contain the backgrounds or black-backgrounds, we do not use the masked loss and let the 3DGS to handle the black-background. If you also add masked loss for synthetic dataset, you will also  find such artifacts. And you can also change the backgrounds of the Stanford-ORB into black, then you can eliminate the artifacts.

### For the differences between synthetic and real-world dataset
I did not observe large discrepancies between synthetic and real-world datasets. Because for both Tenso-IR and Stanford-ORB datasets, they use the similar environment-light to render of Illuminate the object. In Tenso-IR, the authors rendered the images under environment-light map using path-tracing. In Stanford-ORB, the images were captured under studio light, which is similar to environment-light map. 

But for general real-world images, they may captured under various lighting conditons, like point light, direction light, day light or mixture of them, so it is hard to using single environment map to simulate the complex real-world lighting conditions. Then you may find a huge gap between synthetic and real-world data.

**Comment by qbhan** [comment]
Thanks for the tip!
With the mask loss ( I have followed the implementation of R3DG) I could get rid of the artifacts!
![Image](https://github.com/user-attachments/assets/96a60f00-57b5-45f1-b3de-6c7e59177556)

Also, thanks for the insight in real-world settings.
